api-tests/features/repeat-fraud-check/p2-repeat-fraud-check-failure.feature

```diff
@@ -0,0 +1,98 @@
+@Build @RfcFailure
```

Comment:
What are you using these tags for?

I'm not opposed to them, but I think we need a clear and consistent approach - otherwise we'll end up with a load of random tags all over the place.

If you want to run a particular test suite, you can also do that in a couple of other ways:
- `npm run test:local -- features/repeat-fraud-check/p2-repeat-fraud-check-failure.feature` (by file)
- `npm run test:local -- --name 'Given name change'` (by scenario name)

Reply:
Yeah, I guess I'd prefer to group by using files/directories, rather than arbitrary tags - so it helps us navigate the tests when writing them, as well as running them!

Note you can do directories as well, e.g. `npm run test:local -- features/repeat-fraud-check/`

Reply:
OK fair - I'll drop them